aten/src/ATen/native_parse.py

```diff
@@ -71,17 +101,31 @@ def parse_arguments(args, func_decl, func_name, func_return):
         t, name = type_and_name
         default = None
 
+        # Enables Generator? by translating to legacy Generator*. See [temp translations]
+        if t == 'Generator?':
```

Comment:
I think this could be simpler if you wrote another function `temp_default_translations(default, type)` and put all the special logic that isn't in `temp_type_translations` there.

Taking out the translations should basically require no thinking -- just rip out the entry from either  `temp_default_translations(default, type)` or `temp_type_translations`